src/cfnlint/rules/resources/lmbd/SnapStart.py

```diff
@@ -0,0 +1,65 @@
+"""
+Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+SPDX-License-Identifier: MIT-0
+"""
+
+from cfnlint.rules import CloudFormationLintRule, RuleMatch
+
+
+class SnapStart(CloudFormationLintRule):
+    """Check if the settings of multiple subscriptions are included for one LogGroup"""
+
+    id = "W2530"
+    shortdesc = "Validate that SnapStart is properly configured"
+    description = (
+        "To properly leverage SnapStart, you must configure both the lambda function "
+        "and attach a Lambda version resource"
+    )
+    source_url = "https://github.com/awslabs/serverless-application-model/blob/master/versions/2016-10-31.md#user-content-cloudwatchlogs"
+    tags = ["resources", "lambda"]
+
+    def __init__(self):
+        super().__init__()
+        self.resource_property_types.append("AWS::Lambda::Function")
+        self.snapstart_runtimes = ["java11", "java17"]
```

Comment:
As more runtime versions are added this list will need to be maintained. Is there any way to say all Java runtimes except `jvm` and `java8.al2` to save you some work in the future?

Reply:
We can switch this to be anything starting with java that isn't java8.al2